docs/templates/scout-issue-template.md

```diff
@@ -0,0 +1,146 @@
+# Scout Issue Template (Phase 0.0)
+
+Use this template when creating a Pre-Epic Scouting issue to validate feasibility before committing to a full milestone.
+
+---
+
+**Title**: `[Scout] Research {feature-name} feasibility and API patterns`
+
+**Labels**: `research`, `scout`
+
+**Milestone**: (leave empty - no milestone exists yet)
+
+---
+
+## Purpose
+
+Exploratory research to validate feasibility of implementing **{feature-name}** in langstar CLI before committing to a full 8-phase milestone.
+
+## Context
+
+{Brief explanation of why scouting is needed. Examples:}
+- This feature requires API endpoints we haven't used before
+- Unclear if langsmith-sdk Python provides precedent for this functionality
+- Unknown complexity of implementing {specific technical aspect}
+- Need to validate assumptions about API capabilities
+
+## Scope
+
+**Do NOT propose an implementation.** Focus on research only:
+
+1. **Scout Existing Langstar Code**:
+   - Search `./cli` and `./sdk` for partial implementations
+   - Identify any existing types, methods, or patterns that overlap
+   - Document what's already done vs. what's needed
+
+2. **Analyze Python SDK Precedent**:
+   - Use `setup-remote-repo-notes-dir` skill to clone langsmith-sdk
+   - Locate relevant methods in `python/langsmith/client.py`
+   - Document method signatures, parameters, patterns
+   - Review tests for usage examples
+
+3. **Identify API Endpoints**:
+   - Use LangSmith UI or documentation to understand feature behavior
+   - Identify which REST API endpoints are involved
+   - Document request/response shapes (preliminary)
+
+4. **Assess Complexity**:
+   - Rate implementation complexity (low/medium/high)
+   - Identify technical blockers or unknowns
+   - List dependencies on other unimplemented features
+
+5. **Make Go/No-Go Recommendation**:
+   - **Go**: Feature is feasible, proceed to Phase 0 (Epic Setup)
+   - **No-Go**: Feature has blockers, defer or cancel
+   - **Conditional**: Feature is feasible if {conditions met}
```

Comment:
I don't see anything in here about running an experiment, which was a KEY part of certain past milestones such as the structured output prompt ones. that's one of if not the most important things to do when scouting.